indico/modules/events/abstracts/placeholders.py

```diff
@@ -164,7 +164,7 @@ class SubmitterTitlePlaceholder(Placeholder):
 
     @classmethod
     def render(cls, abstract):
-        return abstract.submitter.title
+        return unicode(abstract.submitter.title)
```

Comment:
What about getting the raw string from it instead of translating it using the user's language?

    return orig_string(abstract.submitter.title)